classes/calendar_helpers.php

```diff
@@ -50,13 +50,14 @@ function attendance_create_calendar_event(&$session) {
     $caleventdata->courseid       = $attendance->course;
     $caleventdata->groupid        = $session->groupid;
     $caleventdata->instance       = $session->attendanceid;
-    $caleventdata->timestart      = $session->sessdate;
+    $caleventdata->timestart      = $caleventdata->timesort       = $session->sessdate;
```

Comment:
I'd prefer you match the surrounding code here - one line per variable - can you please drop this onto another line?